src/content/en/updates/2017/06/credential-management-updates.md

```diff
@@ -0,0 +1,333 @@
+
+*  New method `navigator.credentials.create()`
+asynchronously creates credential objects.
+
+### `PasswordCredential` object now includes password 
+
+The Credential Management API took a conservative approach to handling passwords.
+It concealed passwords from JavaScript, requiring developers
+to send the `PasswordCredential` object directly to their server
+for validation via an extension to the `fetch()` API.
+
+But this approach introduced a number of restrictions.
+We received feedback that developers could not use the API because:
+
+* They had to send the password as part of a JSON object.
+
+* They had to send the hash value of the password to their server.
+
+After performing a security analysis and recognizing that concealing passwords
+from JavaScript did not prevent all attack vectors as effectively as we were hoping,
+we have decided to make a change.
+
+The Credential Management API now includes a raw password
+in a returned credential object so you have access to it as plain text.
+You can use existing methods to deliver credential information to your server:
+
+    navigator.credentials.get({
+
+      password: true,
+
+      federated: {
+
+        provider: [ 'https://accounts.google.com' ]
+
+      },
+
+      mediation: 'silent'
+
+    }).then(c => {
+
+      if (c) {
+
+        let form = new FormData();
+
+        form.append('email', c.id);
+
+        form.append('password', c.password);
+
+        form.append('csrf_token', csrf_token);
+
+        return fetch('/signin', {
+
+          method: 'POST',
+
+          credentials: 'include',
+
+          body: form
+
+        });
+
+      } else {
+
+        // Fallback to sign-in form
+
+      }
+
+    }).then(res => {
+
+      if (res.status === 200) {
+
+        return res.json();
+
+      } else {
+
+        throw 'Auth failed';
+
+      }
+
+    }).then(profile => {
+
+      console.log('Auth succeeded', profile);
+
+    });
+
+### Custom fetch will be deprecated soon
+
+Now that passwords return in the `PasswordCredential` object,
+the custom `fetch()` function will stop functioning starting Chrome 62.
+All developers **MUST** make relevant changes to not use a custom `fetch()`.
+
+To determine if you are using a custom `fetch()` function,
+check if it uses a `PasswordCredential` object or a `FederatedCredential` object
+as a value of credentials property and remove it, for example:
+
+    fetch('/signin', {
+
+      method: 'POST',
+
+      **credentials: c**
+
+    })
+
+### `navigator.credentials.get()` now accepts an enum mediation
+
+Until Chrome 60,
+`navigator.credentials.get()` accepted an optional `unmediated` property
+with a boolean flag. For example:
+
+    navigator.credentials.get({
+
+      password: true,
+
+      federated: {
+
+        provider: [ 'https://accounts.google.com' ]
+
+      },
+
+      unmediated: true
+
+    }).then(c => {
+
+      // Sign-in
+
+    });
+
+This positive flag instructs the browser to skip a user mediation.
```

Comment:
This is not simply a "flag", but a flag that is `true`. What is a better expression?